### DISCUSSION TITLE
Should I be able to control branded devices (Idinio)  using the tuya platform?

### DISCUSSION THREAD

**Discussion by jackjansen** [milo526/homebridge-tuya-web#185] [General]
I have some Idinio devices (dutch brand) and these come with an Idinio app. But under the hood this all uses the Tuya platform (as is clear from both the app look-and-feel, and from information on the Tuya website, where they specifically say this is their primary B2B model).

But: it appears homebridge-tuya-web cannot see my devices. I'm attempting to add a device but I get a message in the Homebridge log:

```
[30/01/2021, 23:40:16] [TuyaWebPlatform] Tried overwriting device config for: "99990140d8bfc018be10" which is not a valid device-id or device-name.
```

(actual device-id changed obviously).

Should this work?

**Comment by jackjansen** [comment]
Answering my own question: it turns out that I can access these specific devices through the "smartlife" platform of homebridge-tuya-web. It may be worthwhile to document this somewhere, as I guess more people will have this issue, with all the Tuya-based brands out there....

What I had to do was:
- Unregister the lamp in the Idinio app.
- Download the smartlife app.
- Register in the smartlife app
- the lamp (which was now "factory clean" after removing it from Idinio) could now be added to the smartlife app.
- homebridge-tuya-web could now discover the lamp, and all is well.

**Reply by milo526** [reply]
Glad you figured it all out.
Feel free to submit a Pull Request with improved documentation

**Comment by jackjansen** [comment]
Created pull request #188
